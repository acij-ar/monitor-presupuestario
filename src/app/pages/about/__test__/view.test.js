const React = require('react');
const { shallow } = require('enzyme');

const About = require('../view');

describe('About page', () => {
  it('should match snapshot', () => {
    const props = {
      pageName: 'about',
      "rights": {
        "title": "Presupuesto para los derechos",
        "content": "<p>\n                Las asignaciones presupuestarias tienen implicancias sobre la implementación de políticas públicas\n                destinadas a garantizar la igualdad y superar las condiciones de exclusión de grupos en situación de\n                vulnerabilidad. La conexión inmediata entre el cumplimiento de los derechos y los recursos\n                presupuestarios destinados a realizarlos impone claros límites a las atribuciones estatales sobre el\n                proceso de elaboración de la ley de presupuesto y su ejecución.\n            </p>\n            <p>\n                El acceso a información presupuestaria de manera sencilla, desagregada y accesible, mejora las\n                posibilidades de incidencia e impacto en las acciones tendientes a visibilizar situaciones de\n                discriminación, exclusión y garantía de derechos.\n            </p>\n            <strong>\n                El análisis presupuestario es una herramienta poderosa ya que permite:\n            </strong>\n\n            <ul>\n                <li>\n                    Conocer las diferentes etapas del ciclo presupuestario y los organismos involucrados, los\n                    principales documentos y la información presupuestaria relevante para entender las prioridades de\n                    los gobiernos y sus políticas públicas.\n                </li>\n                <li>\n                    Monitorear la ejecución, las reasignaciones y subejecuciones que ocurren durante el ciclo\n                    presupuestario.\n                </li>\n                <li>\n                    Entablar acciones judiciales y administrativas en reclamo del reconocimiento de derechos\n                    económicos, sociales y culturales, con objeto de poder refutar las excusas alegadas, en la\n                    generalidad de los casos, por el Estado con relación a la escasez de recursos.\n                </li>\n                <li>\n                    Mostrar que en muchos casos esa supuesta falta de recursos enmascara una distribución injusta de\n                    los recursos disponibles, que no son destinados prioritariamente a asegurar el respeto de los\n                    derechos fundamentales.\n                </li>\n            </ul>\n\n            <p>\n                Desde la Asociación Civil por la Igualdad y la Justicia (ACIJ), creamos el <strong>Monitor\n                presupuestario</strong> con el objetivo de promover el acceso a información presupuestaria de manera\n                sencilla, desagregada y accesible, y así contribuir a mejorar las posibilidades de incidencia e\n                impacto en las acciones tendientes a visibilizar situaciones de discriminación, exclusión y garantía\n                de derechos.\n            </p>\n            <p>\n                El <strong>Monitor presupuestario</strong> es una plataforma que contiene información de los diferentes\n                organismos, ministerios, secretarías, programas y actividades presupuestarias desde el año 2007,\n                incluyendo el presupuesto proyectado por el Poder Ejecutivo para 2019.\n            </p>\n            <p>\n                La plataforma permite monitorear la asignación presupuestaria de cada año, el presupuesto aprobado\n                por el Congreso y el presupuesto vigente (con las modificaciones del Poder Ejecutivo), así como la\n                ejecución de años anteriores.\n            </p>"
      },
      "methodology": {
        "title": "Apartado Metodológico",
        "content": "<p>\n                Para desarrollar el <strong>Monitor presupuestario</strong> utilizamos la información provista por el\n                Ministerio de Hacienda, específicamente los dataset de Crédito y Gasto completo (distribución diaria\n                acumulada de los créditos presupuestarios con clasificadores completos) desde el año 2007 al año 2018,\n                disponible en el sitio <a href=\"https://www.presupuestoabierto.gob.ar/sici/\" target=\"_blank\">\n                presupuestoabierto.gob.ar </a>.\n            </p>\n            <p>\n                La información sobre el proyecto de presupuesto 2019 fue solicitada por ACIJ al Ministerio de Hacienda\n                por medio de un pedido de acceso a la información. Asimismo, actualmente se encuentra disponible en el\n                sitio del Ministerio de Hacienda\n                en <a href=\"https://www.minhacienda.gob.ar/onp/presupuesto_ciudadano/index.html\" target=\"_blank\">\n                este link</a>.\n            </p>\n            <strong>Presupuesto 2019</strong>\n            <p>\n                El día 23 de octubre, cuando se trató el Proyecto de Ley de Presupuesto en la Comisión de Presupuesto\n                y hacienda, se emitió un dictamen que modifica el artículo 16 de la Ley, en el cual se establecen las\n                siguientes asignaciones para el ejercicio 2019:\n            </p>\n            <ul>\n                <li>\n                    $ 25.000.000 al Programa 28 Actividad 02 Acciones Inherentes a la Defensa del Consumidor de la\n                    Secretaría de Comercio Interior de la Jurisdicción 51 - Ministerio de Producción y Trabajo con\n                    destino a transferencias a las asociaciones de consumidores según lo determine la autoridad de\n                    aplicación.\n                </li>\n                <li>\n                    $ 900.000.000 para los programas ejecutados por la Secretaría Nacional de la Niñez, Adolescencia y\n                    Familia (SENNAF) de la Jurisdicción 85 - Ministerio de Salud y Desarrollo Social\n                </li>\n                <li>\n                    $ 500.000.000 destinados a la Secretaría de Gobierno de Cultura de la jurisdicción 70 - Ministerio\n                    de Educación, Cultura, Ciencia y Tecnología.\n                </li>\n                <li>\n                    $ 25.000.000 a la actividad 02 del Programa 17; $ 23.000.000 al Programa 43; $ 5.900.000 al Programa\n                    45; $ 6.320.000 al Programa 42; $ 3.600.000 al Programa 22; $ 4.000.000 al Programa 41; $ 28.000.000\n                    al Programa 44; $ 10.000.000 a la Comisión Bicameral Permanente de Fiscalización de los Órganos y\n                    Actividades de Seguridad Interior, ley 24.059 y sus modificatorias, todos ellos pertenecientes a la\n                    Jurisdicción 1 - Poder Legislativo nacional.\n                </li>\n                <li>\n                    $ 30.000.000 a la entidad 918 - Instituto Nacional de las Mujeres destinados al Plan Nacional de\n                    Acción para la Prevención, Asistencia y Erradicación de la Violencia contra las Mujeres,\n                </li>\n                <li>\n                    $ 100.000.000 para el Programa Casas de Atención y Acompañamiento Comunitario (CAACS) dependiente de\n                    la Secretaría de Políticas Integrales sobre Drogas de la Nación Argentina de la Jurisdicción 20.\n                </li>\n                <li>\n                    $ 18.900.000 a la entidad 209 - Agencia de Acceso a la Información Pública.\n                </li>\n                <li>\n                    $ 60.000.000 para la Actividad 01 del Programa 26 de la Jurisdicción 85 con destino a la provincia\n                    de Salta.\n                </li>\n                <li>\n                    $ 70.000.000 al proyecto 25 - Subprograma 01 - Programa 50 - Entidad 604 de la Jurisdicción 57 -\n                    Ministerio de Transporte.\n                </li>\n                <li>\n                    $ 400.000.000 para la Entidad 606 - Instituto Nacional de Tecnología Agropecuaria (INTA).\n                </li>\n                <li>\n                    $ 500.000.000 para la Entidad 103 - Consejo Nacional de Investigaciones Científicas y Tecnológicas\n                    (Conicet).\n                </li>\n                <li>\n                    $ 15.000.000 a la Entidad 101 - Fundación Miguel Lillo.\n                </li>\n                <li>\n                    $ 70.000.000 a la Entidad 804 - Comisión Nacional de Evaluación y Acreditación Universitaria\n                    (CONEAU).\n                </li>\n                <li>\n                    $50.000.000 para el Centro Universitario San Francisco-Córdoba.\n                </li>\n                <li>\n                    $ 550.000.000 a la Entidad 119 - Instituto Nacional de Promoción Turística (Inprotur).\n                </li>\n                <li>\n                    $ 150.000.000 a la Secretaría de Gobierno de Turismo dependiente de la Jurisdicción 20.\n                </li>\n                <li>\n                    $ 3.000.000 para la Congregación Israelita de la República Argentina (CIRA) para ser utilizados en\n                    la puesta en valor de la Sala del Museo Judío de Buenos Aires.\n                </li>\n                <li>\n                    $ 1.000.000 para la Fundación Raíces de Emprendimientos Productivos.\n                </li>\n                <li>\n                    $ 3.373.000.000, con destino al Ministerio del Interior, Obras Públicas y Vivienda.\n                </li>\n            </ul>\n            <strong>Sobre el ajuste por inflación</strong>\n            <p>\n                El análisis del presupuesto, para poder evaluar sus incrementos o disminuciones a lo largo de la\n                gestión, requiere incorporar como variable la tasa de inflación. Para actualizar los montos a precios\n                comparables se debe aplicar la tasa de inflación correspondiente a cada periodo.\n            </p>\n            <p>\n                La serie de precios utilizada surge de los cálculos realizados por EPyCA Consultores, basados en el\n                Índice de Precios al Consumidor (IPC) de Institutos de Estadística Provinciales (hasta marzo de 2016) e\n                IPC Nacional del Instituto Nacional de Estadísticas y Censos (INDEC) (desde marzo de 2016).\n            </p>\n            <p>\n                Los montos del presupuesto para el año en curso se ajustan según el índice de inflación especificado por\n                el Poder Ejecutivo en el proyecto del ley.\n            </p>"
      },
      "information": {
        "title": "Sobre la publicación de información presupuestaria",
        "content": "<p>\n                ACIJ impulsa acciones para promover mejoras en materia de transparencia, participación ciudadana y\n                rendición de cuentas en el proceso presupuestario bajo la concepción de que el presupuesto es una\n                herramienta clave para discutir la asignación de recursos adecuados para las políticas públicas\n                destinadas a garantizar los derechos de los grupos vulnerabilizados.\n            </p>\n\n            <strong>Transparencia presupuestaria</strong>\n            <p>\n                Para analizar los presupuestos con un enfoque de derechos humanos, resulta clave acceder oportunamente a\n                información completa y adecuadamente desagregada sobre los recursos asignados y ejecutados para la\n                realización de cada una de las actividades que desarrolla el Estado, a través de la administración\n                centralizada y descentralizada, así como comprender adecuadamente en qué consisten esas actividades.\n            </p>\n\n            <p>\n                A su vez, para relacionar la asignación y los recursos efectivamente utilizados con la realización o\n                violación de derechos fundamentales también resulta indispensable acceder a información adecuada en\n                relación con indicadores estadísticos sociales y económicos desagregados, que permitirán evaluar en\n                forma concreta el impacto del gasto sobre los niveles de cumplimiento de los derechos.\n            </p>\n\n            <p>\n                La falta de acceso oportuno a información presupuestaria adecuadamente desagregada es un desafío\n                constante que debemos enfrentar, pero de manera reciente se han producido mejoras significativas en la\n                cantidad de la información accesible de manera pública, en relación con la asignación y ejecución de la\n                Administración centralizada.\n            </p>\n\n            <p>\n                Si bien aún resta mucha información por abrir para asegurar una verdadera transparencia presupuestaria,\n                el trabajo constante de organizaciones de la sociedad civil ha generado avances concretos en la cantidad\n                de información disponible, y en la sanción de normas claras que establecen la obligación estatal de\n                publicar proactivamente la información hasta el máximo nivel de desagregación.\n            </p>\n\n            <p>\n                A modo de ejemplo, anteriormente no se publicaba información sobre la asignación y ejecución que\n                estuviera desagregada al nivel de “actividad”, ya que sólo se proporcionaba información hasta el nivel\n                de “programa”. Los programas estatales involucran la realización de diversas actividades, cuyo detalle\n                es fundamental para conocer de manera concreta y real, exactamente a qué acciones concretas y puntuales\n                se destinan los recursos. A su vez, tampoco se podía acceder a la información disponible en formato de\n                datos abiertos, lo que dificultaba enormemente la utilización y procesamiento de la información. También\n                era extremadamente limitada la información disponible sobre metas físicas, es decir, sobre los bienes y\n                servicios que se producen con la realización de las actividades estatales, y la información\n                presupuestaria disponible era actualizada con una periodicidad mucho mayor, con lo cual su acceso no era\n                realmente oportuno.\n            </p>\n\n            <p>\n                El Ministerio de Hacienda publica datasets en formato de datos abiertos, que permiten acceder a\n                información actualizada con una frecuencia de 48 horas y desagregada hasta el nivel de actividad,\n                clasificada por tipo de gasto, finalidad-función, objeto del gasto, fuente de financiamiento, por\n                naturaleza económica, etc.\n            </p>\n\n            <p>\n                Los mayores niveles de apertura de datos hacen posible su utilización efectiva por la sociedad civil,\n                para participar de la discusión presupuestaria, e incidir sobre las prioridades estatales en la\n                asignación y ejecución de los recursos públicos.\n            </p>\n\n            <p>\n                Corresponde aclarar, a su vez, que esta apertura de datos aún tiene limitaciones. En efecto, la apertura\n                de datos no alcanza a todos los organismos de la administración descentralizada, en muchos casos aún\n                falta información debidamente detallada sobre algunas actividades, no se publica información sobre\n                beneficiarios/as de cada una de las transferencias de recursos estatales, como pagos a proveedores,\n                beneficiarios/as de subsidios, entre otros supuestos. A su vez, si bien desde hace varios años se están\n                generando algunos avances en la generación de indicadores de impacto concreto de los programas, todavía\n                resta la definición de indicadores adecuados en muchos programas estatales. Tampoco es posible acceder\n                al presupuesto original en datos abiertos sino solo a los presupuestos inicial, vigente y devengado.\n            </p>"
      },
      "dictionary": {
        "title": "Glosario",
        "content": "<p>\n                <strong>Actividades</strong>: son las categorías en que se distribuye el presupuesto dentro de los\n                diferentes programas presupuestarios específicos.\n</p>\n            <p>\n                <strong>Ciclo presupuestario</strong>: comprende las etapas de formulación, aprobación, ejecución y\n                auditoría o monitoreo del presupuesto.\n            </p>\n            <p>\n                <strong>Distribución administrativa</strong>: consiste en la presentación desagregada de los créditos y\n                realizaciones contenidas en la Ley de Presupuesto. Una vez aprobada la Ley de Presupuesto, el Poder\n                Ejecutivo emite una Decisión Administrativa donde distribuye los gastos corrientes y de capital, los\n                gastos figurativos, las aplicaciones financieras, los recursos, las contribuciones figurativas y las\n                fuentes de financiamiento previstas en la Ley de Presupuesto General de la Administración Nacional del\n                ejercicio en curso.\n            </p>\n            <p>\n                <strong>Jurisdicciones</strong>: son organizaciones públicas sin personalidad jurídica ni patrimonio\n                propio, que integran la Administración Central y representan los Poderes (Legislativo, Judicial y\n                Ejecutivo) establecidos por la Constitución Nacional. Algunas de ellas son: Ministerio Público,\n                Presidencia de la Nación, Jefatura de Gabinete de Ministros, Ministerios y Secretarías del Poder\n                Ejecutivo Nacional.\n            </p>\n            <p>\n                <strong>Metas físicas</strong>: es la medición física, en términos de bienes y servicios, de los\n                resultados que se persiguen con los programas presupuestarios. Por ejemplo, la cantidad de libros que se\n                repartirán, la cantidad de viandas y becas que se otorgarán, etc.\n            </p>\n            <p>\n                <strong>Presupuesto devengado</strong>: nivel de gastos efectivamente gastados o ejecutados a una\n                determinada fecha. El porcentaje de ejecución mide el grado de avance de la ejecución de los gastos\n                sobre el nivel del presupuesto vigente.\n            </p>\n            <p>\n                <strong>Presupuesto inicial</strong>: crédito distribuido por el Poder Ejecutivo al inicio del ciclo\n                presupuestario.\n            </p>\n            <p>\n                <strong>Presupuesto original</strong>: crédito aprobado por el Congreso en la Ley de Presupuesto del\n                correspondiente ejercicio.\n            </p>\n            <p>\n                <strong>Presupuesto vigente</strong>: límite máximo disponible para gastar en el momento en que se\n                presenta determinada información, independientemente de lo que haya sido aprobado al inicio del año (si\n                el presupuesto no tuvo modificaciones durante el año, el crédito vigente coincidirá con el original)\n                porque incorpora las modificaciones presupuestarias que afectan el presupuesto original en más o en\n                menos.\n            </p>\n            <p>\n                <strong>Programas</strong>: categoría de programación que es unidad formal de asignación de recursos y\n                tiene el propósito de satisfacer demandas de la comunidad en el contexto de una política presupuestaria\n                específica.\n            </p>\n            <p>\n                <strong>Reasignación</strong>: cambio en el destino de fondos que se habían asignado para determinado\n                fin en la Ley de Presupuesto aprobada por el Congreso.\n            </p>\n            <p>\n                <strong>Subejecución presupuestaria</strong>: utilización de un monto de dinero inferior al disponible\n                para gastar. Normalmente es calculado como la diferencia entre el crédito vigente y el monto devengado.\n                Los montos subejecutados también suelen ser identificados como “saldos no utilizados”.\n            </p>"
      }
    };
    const wrapper = shallow(<About {...props}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
