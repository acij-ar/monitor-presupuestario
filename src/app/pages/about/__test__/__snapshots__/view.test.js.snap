// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page should match snapshot 1`] = `
<Page
  pageName="about"
>
  Acerca de
</Page>
`;
